src/components/Button.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const defaultColor = 'rgb(249, 144, 77)'
+function Button (props) {
+  const style = {
+    width: props.wide ? '50%' : '25%',
+    background: props.color ? props.color : defaultColor
+  }
+  return (
+    <button style={style} onClick={() => props.clickHandler(props.name)}>{props.name}</button>
+  )
+}
+
+Button.prototype = {
```

Comment:
This should be `Button.propTypes`, don't you think?

Also, it's not:

```javascript
name: PropTypes.String
```

It's supposed to be:

```javascript
name: PropTypes.string.isRequired
```

Let us review how to use prop-type validations from this [article](https://blog.logrocket.com/validating-react-component-props-with-prop-types-ef14b29963fc/)